go.mod

```diff
@@ -4,17 +4,24 @@ go 1.13
 
 require (
 	github.com/go-logr/logr v0.1.0
+	github.com/onsi/ginkgo v1.12.0
+	github.com/onsi/gomega v1.9.0
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
 	github.com/openshift/cluster-image-registry-operator v0.0.0-20200415091009-99c06ee64540
 	github.com/openshift/custom-resource-status v0.0.0-20190822192428-e62f2f3b79f3
+	github.com/openshift/ocs-operator v0.0.1-alpha1
```

Comment:
We shouldn't be pulling cos-operator in.

Reply:
Sorry, I didn't pay enough attention to these 2 files. Thanks for catching :eyes: